apps/web/src/views/Swap/MMLinkPools/components/MMCommitButton.tsx

```diff
@@ -71,10 +65,11 @@ export function MMSwapCommitButton({
   const { t } = useTranslation()
   // the callback to execute the swap
 
-  const swapCalls = useSwapCallArguments(rfqTrade.trade, rfqTrade.rfq, recipient)
+  // @ts-ignore
+  const swapCalls = useSwapCallArguments(rfqTrade?.trade, rfqTrade?.rfq as RFQResponse['message'], recipient)
```

Comment:
Let's update the input type of the hook to accept undefined as input

Reply:
so no need to do type assertion here

Reply:
done